### DISCUSSION TITLE
[BUG] Connection issue

### DISCUSSION THREAD

**Discussion by ptrinephi** [cagnulein/qdomyos-zwift#292] [Q&A]
2.6.9 it conencts then disconnects, this is a new android 9 device, can you check what the issue is?  Thanks

[debug-Sun_Apr_18_13_36_07_2021.log](https://github.com/cagnulein/qdomyos-zwift/files/6332290/debug-Sun_Apr_18_13_36_07_2021.log)
[debug-Sun_Apr_18_13_38_16_2021.log](https://github.com/cagnulein/qdomyos-zwift/files/6332291/debug-Sun_Apr_18_13_38_16_2021.log), 
[debug-Sun_Apr_18_13_38_55_2021.log](https://github.com/cagnulein/qdomyos-zwift/files/6332292/debug-Sun_Apr_18_13_38_55_2021.log)can s
[Uploading debug-Sun_Apr_18_13_41_32_2021.log…](y
[debug-Sun_Apr_18_13_46_20_2021.log](https://github.com/cagnulein/qdomyos-zwift/files/6332294/debug-Sun_Apr_18_13_46_20_2021.log)ou 
[debug-Sun_Apr_18_13_50_41_2021.log](https://github.com/cagnulein/qdomyos-zwift/files/6332295/debug-Sun_Apr_18_13_50_41_2021.log)ce
[debug-Sun_Apr_18_13_53_07_2021.log](https://github.com/cagnulein/qdomyos-zwift/files/6332296/debug-Sun_Apr_18_13_53_07_2021.log)hck
[debug-Sun_Apr_18_13_54_44_2021.log](https://github.com/cagnulein/qdomyos-zwift/files/6332297/debug-Sun_Apr_18_13_54_44_2021.log)
[debug-Sun_Apr_18_13_56_49_2021.log](https://github.com/cagnulein/qdomyos-zwift/files/6332298/debug-Sun_Apr_18_13_56_49_2021.log)

[debug-Sun_Apr_18_13_34_46_2021.log](https://github.com/cagnulein/qdomyos-zwift/files/6332289/debug-Sun_Apr_18_13_34_46_2021.log)

**Comment by cagnulein** [comment]
You're always the king of the strange problems :)
It seems that the bike close the connection immediately.
Did you pair the bike into the bluetooth settings? if so unpair it.
Does the echelon app work on this phone?

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno lun 19 apr 2021 alle ore 00:04 3phi ***@***.***> ha
scritto:

**Comment by ptrinephi** [comment]
Lol, You seem like the kind of guy who likes a challenge so I'm just 
having fun testing things for you, hehe ;-)

This is in fact running on the echelon tablet itself. The odd thing is 
it sometimes works for a few minutes then disconnects but the bike's 
connected blue light stays on even after turning off the tablet.  I have 
to power cycle the bike to bring it back to normal.

BTW, I installed QT and tried to build your app hoping to help you debug 
a few things but I was not able to build the project.  Android dev is 
really new to me...!

------ Original Message ------

**Comment by cagnulein** [comment]
I guess they put some customized bluetooth stack on it

Il lun 19 apr 2021, 18:06 3phi ***@***.***> ha scritto:

**Comment by ptrinephi** [comment]
probably!  Would there be a way to find out and bypass it?

------ Original Message ------

**Comment by cagnulein** [comment]
No idea :D
